app/controllers/api/v1/Web.java

```diff
@@ -2008,4 +2009,35 @@ private static long parseTime(String time) {
     }
     return unixTime;
   }
+
+  /**
+   * Parse the JobId to extract the cluster name
+   *
+   * @param jobId JobId
+   * @return the cluster name if the jobId belongs to configured clusters else None
+   */
+  private static String getClusterName(String jobId) {
+    if (azkabanKeyToClusterNameMap.size() == 0) {
+      if (System.getProperty(AZKABAN_TO_CLUSTER_MAP_KEY) == null) {
+        logger.error("Azkaban to Cluster Map is not present in the config file");
+        return "None";
+      }
+      logger.info("Initializing the azkaban_to_cluster_name map");
+      String[] azkabanClusterList = System.getProperty("azkaban_to_cluster_map").split(",");
+      for (String cluster : azkabanClusterList) {
+        String[] row = cluster.split(":");
+        if (row.length != 2) {
```

Comment:
please create enum with expected schema after cluster.split. and use those enum ordinal , instead of directly accessing index